### PR TITLE
fix: remove SelectInput rerender for options live update

### PR DESCRIPTION
**Pull request by petror-nulogy**
## Description

**Describe the change you're making, the motivations behind it, and any thing else a reviewer may need to know to approve this PR.**

## Changes include

- [ ] breaking change: a change that is not backwards-compatible and/or changes current functionality
- [x] fix: a non-breaking change that solves an issue
- [ ] feature: a non-breaking change that adds functionality
- [ ] chore: contains no changes affecting the library, such as documentation or test updates

## Feature checklist

- [ ] Appropriate tests have been added 
- [ ] Documentation has been updated
- [ ] Accessibility has been considered

### CODE REVIEW

File: cypress/integration/components/Select.spec.js [hunk 1]
```diff
@@ -136,6 +136,39 @@ describe("Select", () => {
     });
   });
 
+  describe.only("Multiselect: options live update", () => {
```
Comment (haideralsh) [inline cypress/integration/components/Select.spec.js]: I think we need to remove the only here.

File: cypress/integration/components/Select.spec.js [hunk 2]
```diff
@@ -136,6 +136,39 @@ describe("Select", () => {
...
+
+    it("add new option on every input change and input should be focused", () => {
+      getSelectTextInput().type("test");
+      getSelectTextInput().should("have.focus");
+
+      const csvValues = "PCN2, PCN1";
+
+      cy.paste({ destinationSelector: selectTextInputSelector, pastePayload: csvValues });
+
+      cy.focused().type("{backspace}{backspace}{backspace}");
```
Comment (haideralsh) [inline cypress/integration/components/Select.spec.js]: Why do we need to backspace before asserting if the input is focused?

File: cypress/integration/components/Select.spec.js [hunk 3]
```diff
@@ -136,6 +136,39 @@ describe("Select", () => {
...
+      getSelectTextInput().should("have.focus");
+
+      cy.focused().type("{enter}");
+      cy.focused().type("{enter}");
+    });
+
+    it("user able to paste and select values with live update of options", () => {
```
Comment (haideralsh) [inline cypress/integration/components/Select.spec.js]: This test seems very similar to the one above, where we type and assert focus. Is there anything else that it is testing?

Comment (petror-nulogy) [reply cypress/integration/components/Select.spec.js]: Yes, good point.
I've rethought this case and left only one test case.

File: cypress/integration/components/Select.spec.js (hunk 1 above)
Comment (petror-nulogy) [reply cypress/integration/components/Select.spec.js]: Thank you! Good catch.

**Review by haideralsh — APPROVED** [review]

**Comment by haideralsh** [comment]
We can merge this now, and come back to refactor the tests.

**Pull request merged**